### ISSUE TITLE
Unable to get Vorlon working on Outlook 2016 for Mac

### ISSUE THREAD

**Issue by abhishek1234321** [MicrosoftDX/Vorlonjs#387]
OSX Version: Sierra
Outlook 2016

I have included the `vorlon.js` file in my addin, when trying to run the addin from Outlook Desktop Client, it doesn't get detected in Vorlon Dashboard.

But when I open it directly in the browser, it does.

Is there a way to make it detect directly from the desktop client?

**Comment by meulta** [comment]
Did you follow this blog article talking about the certificate? https://blogs.msdn.microsoft.com/mim/2016/02/18/vorlonjs-plugin-for-debugging-office-addin/

**Comment by abhishek1234321** [comment]
Yup, I did.. all steps, unfortunately it doesn't seem to work, is there anyway to identify what is wrong?

**Comment by abhishek1234321** [comment]
Unable to load `https://localhost:1337` in Safari, works fine in Chrome though..

**Comment by meulta** [comment]
Ah! Can you go in the console et see if there are any errors in safari?

**Comment by mosspaper** [comment]
I was experience the same issue and was able to get Vorlon working with Outlook 2016 for Mac by following this post:

https://dev.office.com/docs/add-ins/testing/debug-office-add-ins-on-ipad-and-mac

**Comment by philjones88** [comment]
I was having this too.

My steps to resolve on OSX + Outlook 2016 for Mac.

1. Open safari
2. Go to https://localhost:1337
3. When the certificate warning pops up, click always trust/allow etc
4. Reload your addin (right click, reload).
5. It works!

**Comment by draisy** [comment]
I'm facing this problem with my Excel 2016 Add-in on Mac OSX El Capital, as well. I was able to get Vorlon running just one time, but after restarting Vorlon, it is no longer identifying any client in the dashboard. My scripts are all in place, my ssl settings are set, and I've tried all the steps above but my add-in is no longer being detected. 

Any additional insight?
